tests/test_tokenization_tapas.py

```diff
@@ -0,0 +1,1195 @@
+                5187,
+                2340,
+                2281,
+                3326,
+                2577,
+                18856,
+                7828,
+                3240,
+                5354,
+                6353,
+                1020,
+                2089,
+                3777,
+            ],
+            "attention_mask": [
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+            ],
+            "token_type_ids": [
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [1, 1, 0, 0, 0, 0, 0],
+                [1, 2, 0, 0, 0, 0, 0],
+                [1, 3, 0, 0, 0, 0, 0],
+                [1, 3, 0, 0, 0, 0, 0],
+                [1, 3, 0, 0, 0, 0, 0],
+                [1, 4, 0, 0, 0, 0, 0],
+                [1, 4, 0, 0, 0, 0, 0],
+                [1, 4, 0, 0, 0, 0, 0],
+                [1, 1, 1, 0, 0, 0, 0],
+                [1, 1, 1, 0, 0, 0, 0],
+                [1, 2, 1, 0, 2, 2, 0],
+                [1, 3, 1, 0, 3, 1, 0],
+                [1, 4, 1, 0, 2, 2, 0],
+                [1, 4, 1, 0, 2, 2, 0],
+                [1, 4, 1, 0, 2, 2, 0],
+                [1, 1, 2, 0, 0, 0, 0],
+                [1, 1, 2, 0, 0, 0, 0],
+                [1, 1, 2, 0, 0, 0, 0],
+                [1, 1, 2, 0, 0, 0, 0],
+                [1, 2, 2, 0, 1, 3, 0],
+                [1, 3, 2, 0, 1, 3, 0],
+                [1, 4, 2, 0, 3, 1, 0],
+                [1, 4, 2, 0, 3, 1, 0],
+                [1, 4, 2, 0, 3, 1, 0],
+                [1, 1, 3, 0, 0, 0, 0],
+                [1, 1, 3, 0, 0, 0, 0],
+                [1, 1, 3, 0, 0, 0, 0],
+                [1, 1, 3, 0, 0, 0, 0],
+                [1, 2, 3, 0, 3, 1, 0],
+                [1, 3, 3, 0, 2, 2, 0],
+                [1, 4, 3, 0, 1, 3, 0],
+                [1, 4, 3, 0, 1, 3, 0],
+                [1, 4, 3, 0, 1, 3, 0],
+            ],
+        }
+
+        new_encoded_inputs = tokenizer.encode_plus(table=table, query=queries[0], padding="max_length")
```

Comment:
This test is to prepare inputs for TAPAS for inference. We also need to include tests to prepare inputs for TAPAS for training (fine-tuning). In that case, also `answer_coordinates` and `answer_text` need to be provided to the tokenizer, and it should create `label_ids`, `numeric_values` and `numeric_values_scale`.